types/got/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for got 8.3
+// Type definitions for got 9.2
```

Comment:
I think need to create separate declaration file for v8 (like in declarations for node.js), because `got v9` works only with `nodejs >= 8` and we should can update previous got version somehow.